### ISSUE TITLE
Feature Request: Sort imap mail by oldest first.

### ISSUE THREAD

**Issue by sn3ak** [antedebaas/DMARC-Reports#44]
I've saved my DMARC and TLS reports for years as I didn't have software to view the reports. Having found your software I set it up and had it parse the 300+ emails that were saved. The result isn't what I would have expected, as "Latest DMARC/TLS Reports" shows the oldest email first.

While looking around  I stumbled on the following code from php's [imap_fetch_overview](https://www.php.net/manual/en/function.imap-fetch-overview.php#116727) page. I am wondering if you could implement something similar
```
<?php
$result = imap_fetch_overview($imap_stream, '1:10', 0);

usort($result, function($a, $b) {
  return($b->udate-$a->udate);
});
?> 
```

**Comment by antedebaas** [comment]
im not using imap functions directly but the php-imap library.
im not sure if that supports a sorting mechanism and ill have to look into that

**Comment by sn3ak** [comment]
I researched this some more and found the following answer from the sources [github](https://github.com/barbushin/php-imap/issues/470):
It appears adding `rsort($mail_ids);` before line [87](https://github.com/antedebaas/DMARC-SMTPTLS-Reports/blob/ecf7e0930bcc6987908290d0ccad47f81a211b9c/src/Command/GetReportsFromMailboxCommand.php#L87) will handle this.

**Comment by sn3ak** [comment]
I dropped my database, recreated it and ran `php bin/console doctrine:migrations:migrate --no-interaction --query-time` then visited http://127.0.0.1/setup to create a user and finish the install. I then used webmail to set all messages as UNSEEN.

I ran the script to fetch mail, and besides the errors I am getting in [#45 ](https://github.com/antedebaas/DMARC-SMTPTLS-Reports/issues/45), I was able to import all reports, and they all appear to be in the correct order now after making the rsort change.
